### DISCUSSION TITLE
How to indent lines?

### DISCUSSION THREAD

**Discussion by lilaque** [knowledgefutures/pubpub#2722] [Support]
Hi all--

Is there a command or sequence of keys to press that will let me indent lines? I searched the help page without success.

I would like to indent lines in a PubPub document without having to make a formatted or unformatted list--like you would do in Word with the Tab key.

I have tried using the Tab key, but it toggles my position within the document.

**Comment by MacineP** [comment]
Try holding Shift+Tab after selecting the code.

**Comment by jt-notes** [comment]
I also would like this. Some authors like to divide things using indentation.
